backend/src/services/group/group.service.ts

```diff
@@ -47,9 +49,20 @@ class Group {
       permissionsIds,
     });
 
-    const newGroup = await this.#groupRepository.create(group);
+    return this.#groupRepository.create(group);
+  }
+
+  public async delete({ id }: EAMGroupDeleteParamsDto): Promise<number> {
+    const group = await this.#groupRepository.getGroupById(id);
+
+    if (group!.users.length >= 1) {
```

Comment:
how to get rid of not-null assertion here?

Reply:
cast type `getGroupById ` should return an object with `users ` 👍

Reply:
```
if (group === null) throw new EamError()

if (group.users.length >= 1) ...
```